include/libtorrent/torrent_status.hpp

```diff
@@ -78,7 +78,12 @@ TORRENT_VERSION_NAMESPACE_2
 		// only the torrent_handle field is compared.
 		bool operator==(torrent_status const& st) const
 		{ return handle == st.handle; }
-
+	
+		bool is_save_resume() const
+		{
+		  return handle.is_valid() && has_metadata && need_save_resume;
```

Comment:
Are you taking the addresses of these member functions and converting them to bools?

Reply:
I think this should be a free function, and be defined in client_test.cpp

Reply:
I don't understand the naming here either. what does it refer to, that *is* save resume?
It seems it would make much more sense for the function to be called: `need_save_resume()`

Reply:
as far as I can tell, `torrent_status::need_save_resume` should be deprecated. This state is part of the `flags` now.

Reply:
Do you disagree with this being a free function?